Wish list

ayanjamil opened the pull request:
tested Wish list page on Pixel 7 ( Screen Size = 6.31 inch ) , Pixel 3a( Screen Size = 5.6 inch) , Pixel 2(Screen Size = 5.0 inch) and Pixel 7 Pro ( Screen Size = 6.71 inch)
![Screenshot from 2024-01-18 14-52-53](https://github.com/udaybansal19/Shimi/assets/102759044/3195f9fc-e18a-4210-a37d-2a16eeffe323)
![Screenshot from 2024-01-18 14-46-11](https://github.com/udaybansal19/Shimi/assets/102759044/cbb2e183-6bc4-4ad7-bdc2-3118fc7b3e8
![Screenshot from 2024-01-16 22-36-44](https://github.com/udaybansal19/Shimi/assets/102759044/8418271c-8e05-4746-960c-0c1828c33eb2)
e)
![Screenshot from 2024-01-18 14-36-39](https://github.com/udaybansal19/Shimi/assets/102759044/92008a8c-1bd7-4756-b9a5-a7dca00a3465)

udaybansal19 commented:
@ayanjamil you can check in the screenshot that the distance between the product image and the view button is different on different screens. It should be constant across screens according to design

udaybansal19 commented on components/CardWishList.js:
```diff
@@ -73,73 +65,71 @@ CardWishList.propTypes = {
+  buttons: { 
+    flexDirection: "row",
+    alignItems: "center",  
+    marginTop:'17%',
```
margins should me absolute

udaybansal19 commented on components/CardWishList.js:
```diff
@@ -73,73 +65,71 @@ CardWishList.propTypes = {
-    marginRight:60,  
+  viewBtn: {
+    backgroundColor:'#DD4A65',
+    borderRadius:20,
```
incorrect linting

udaybansal19 commented on components/CardWishList.js:
```diff
@@ -47,23 +30,32 @@ class CardWishList extends React.Component {
+                </Button> 
+                <TouchableWithoutFeedback onPress={() => navigation.navigate('Pro')}>
+                  <Block flex >
+                    <Image source={require("../assets/imgs/bin.png")} style={styles.logo} /> 
```
check for trash icon in icon libraries

udaybansal19 commented on screens/WishList.js:
```diff
@@ -5,41 +5,26 @@ import {
-  };
-
   renderCards = () => {
     var wishlistCards = [];
```
create a constant array and loop over the elements using .map.

ayanjamil commented:
•  Top Navigation bar 
• Counter  for the number of wishlist items
• Formated code 
• Fixed padding and margin values 
• Added usable image files in constants and used in the home screen.
![Screenshot from 2024-01-21 19-29-00](https://github.com/udaybansal19/Shimi/assets/102759044/2531ed36-7e41-463c-ac85-34d9158bccd5)
![Screenshot from 2024-01-21 19-21-57](https://github.com/udaybansal19/Shimi/assets/102759044/afcb99fe-9d18-47ed-95c9-f849ad42de69)


![Screenshot from 2024-01-21 19-29-55](https://github.com/udaybansal19/Shimi/assets/102759044/5451bc62-2c7f-47c3-94b1-eef1f77e85e9)

udaybansal19 commented on .vscode/settings.json:
```diff
@@ -0,0 +1,4 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```
this file shouldn't not be committed. add it to .gitignore if needed

udaybansal19 commented on assets/imgs/def.png:
these files can be removed

udaybansal19 commented on components/CardWishList.js:
```diff
@@ -0,0 +1,154 @@
+        <Block>
+          <Image
+            source={{
+              uri: "https://encrypted-tbn2.gstatic.com/favicon-tbn?q=tbn:ANd9GcSrR7Ml_19H_B-zDu_Z_bQZGw0QqQ9U4wrcsyIsLYTUdUbRnR8dQRl2hFbwwMTnrsx4dn8RSieDAX0xgOMr90CKQT2BqGauj6jT9xLf-SQOorNViw",
```
this should be sent throught props

udaybansal19 commented on components/CardWishList.js:
```diff
@@ -0,0 +1,154 @@
+        </Block>
+        <Block>
+          <Text size={15} style={{ color: "#A6A6A6", marginHorizontal: "7%" }}>
+            Mango
```
this as well

udaybansal19 commented on components/CardWishList.js:
```diff
@@ -0,0 +1,154 @@
+              >
+                <Block flex style={styles.trash}>
+                  {/* <Image
+                    source={require("../assets/imgs/bin.png")}
```
please remove unused code

udaybansal19 commented on components/Header.js:
```diff
@@ -163,83 +264,87 @@ class Header extends React.Component {
-        {this.renderHeader()}
-      </Block>
-    );
+    if (title != "home") {
```
why is this check needed?

udaybansal19 commented on screens/WishList.js:
```diff
@@ -0,0 +1,80 @@
+  }
+  renderCards = () => {
+    var wishlistCards = [];
+    for (let i = 0; i < this.state.itemCount; i++) {
```
use `usables.map` to display all items from constants array

udaybansal19 commented on screens/WishList.js:
```diff
@@ -0,0 +1,75 @@
+    };
+  }
+  componentDidMount() {
+    this.setState({ itemCount: 23 });
```
this is not needed now
